cdc/model/owner.go

```diff
@@ -238,6 +240,7 @@ func (ts *TaskStatus) RemoveTable(id TableID, boundaryTs Ts, isMoveTable bool) (
 		return nil, false
 	}
 	delete(ts.Tables, id)
+	log.Info("remove a table", zap.Int64("table-id", id), zap.Uint64("boundaryTs", boundaryTs), zap.Bool("isMoveTable", isMoveTable))
```

Comment:
how about rename `table-id` to `tableID`, to keep log format pattern the same, just like `isMoveTable` or `ResolvedTs`

Reply:
done